Apps/phdossier/dossier_connector.py

```diff
@@ -0,0 +1,342 @@
+            summary = action_result.update_summary({})
+            if response.get("results"):
+                if response["results"][0]["data"].get("details", {}).get("av_match_count"):
+                    summary['results'] = response["results"][0]["data"].get("details", {}).get("av_match_count")
+                else:
+                    summary['results'] = 0
+
+            # Return success, no need to set the message, only the status
+            # BaseConnector will create a textual message based off of the summary dictionary
+            return action_result.set_status(phantom.APP_SUCCESS)
+        else:
+            return action_result.set_status(phantom.APP_ERROR, "Error fetching data")
+
+    def _handle_lookup_url(self, param):
+
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+
+        # Add an action result object to self (BaseConnector) to represent the action for this param
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # Access action parameters passed in the 'param' dictionary
+
+        # Required values can be accessed directly
+        # submitted_url comes from phantom. not the best name, i know.
+        submitted_url = param['url']
+
+        # format our url
+        url = "services/intel/lookup/indicator/url?value={}&source=atp&wait=true".format(submitted_url)
+
+        # make rest call
+        response, status_code = self._make_rest_call(url)
+
+        if status_code == 200:
+            # Add the response into the data section
+            action_result.add_data(response["results"])
+
+            # Add a dictionary that is made up of the most important values from data into the summary
+            summary = action_result.update_summary({})
+            if response.get("results"):
+                summary['results'] = response["results"][0]["data"]["record_count"]
+
+            # Return success, no need to set the message, only the status
+            # BaseConnector will create a textual message based off of the summary dictionary
+            return action_result.set_status(phantom.APP_SUCCESS)
+        else:
+            return action_result.set_status(phantom.APP_ERROR, "Error fetching data")
+
+    def _handle_lookup_ip(self, param):
+
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+
+        # Add an action result object to self (BaseConnector) to represent the action for this param
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # Access action parameters passed in the 'param' dictionary
+
+        # Required values can be accessed directly
+        ip = param['ip']
+
+        # format our url
+        url = "services/intel/lookup/indicator/ip?value={}&source=atp&wait=true".format(ip)
+
+        # make rest call
+        response, status_code = self._make_rest_call(url)
+        print(response["results"])
```

Comment:
Either take this out, add it as progress, or add it as debug

Reply:
Fixed.